website_docs/getting-started.md

```diff
@@ -1,12 +1,13 @@
 ---
-date: '2021-05-07T21:49:47.106Z'
+date: '2021-08-30T15:17:00.886Z'
 docname: getting-started
 images: {}
 path: /getting-started
-title: "Getting Started"
-weight: 22
+title: Getting Started with OpenTelemetry Python
```

Comment:
Will this interfere with the sidebar navigation on the website?

<img width="200" alt="Screen Shot 2021-08-30 at 9 17 00 AM" src="https://user-images.githubusercontent.com/223565/131370937-8dfd63d4-7131-40ce-b2db-c6656cb7c90b.png">

Reply:
Good catch. It is under Python section anyway so "with OpenTelemetry Python" is very redundant.